roles/installer/templates/config.yaml.j2

```diff
@@ -30,6 +30,13 @@ data:
     JOBOUTPUT_ROOT = '/var/lib/awx/job_status'
 
     IS_K8S = True
+    SYSTEM_TASK_ABS_MEM = ''
+    task_resource_requirements = '{{ task_resource_requirements }}'
```

Comment:
this probably need some filter to get the `task_resource_requirements` to render to valid python dictionary